docs/Development/Start new OpenY project.md

```diff
@@ -3,23 +3,84 @@ Start new OpenY project
+
+2. Add all custom repositories that are [listed here](https://github.com/ymcatwincities/openy-project/blob/8.1.x/composer.json#L31) to your `composer.json`
+
+3. Add installer path as here to your `composer json`. See [example](https://github.com/ymcatwincities/openy-project/blob/8.1.x/composer.json#L165).
+
+- `composer.json` **inside** of docroot
+Installer path will look like this:
+
+    ```
+    "installer-paths": {
+        "core": ["type:drupal-core"],
+        "libraries/{$name}": ["type:drupal-library"],
+        "modules/contrib/{$name}": ["type:drupal-module"],
+        "profiles/contrib/{$name}": ["type:drupal-profile"],
+        "themes/contrib/{$name}": ["type:drupal-theme"],
+        "drush/contrib/{$name}": ["type:drupal-drush"],
+        "modules/custom/{$name}": ["type:drupal-custom-module"],
+        "themes/custom/{$name}": ["type:drupal-custom-theme"]
+    }
+     ```
+
+- `composer.json` **outside** of docroot
+Installer path will look like this:
+
+    ```
+    "installer-paths": {
+        "docroot/core": ["type:drupal-core"],
+        "docroot/libraries/{$name}": ["type:drupal-library"],
+        "docroot/modules/contrib/{$name}": ["type:drupal-module"],
+        "docroot/profiles/contrib/{$name}": ["type:drupal-profile"],
+        "docroot/themes/contrib/{$name}": ["type:drupal-theme"],
+        "drush/contrib/{$name}": ["type:drupal-drush"],
+        "docroot/modules/custom/{$name}": ["type:drupal-custom-module"],
+        "docroot/themes/custom/{$name}": ["type:drupal-custom-theme"]
+    }
+    ```
+
+4. Add `"cweagans/composer-patches": "~1.0"` to the `require` section in you `composer.json`. See [example](https://github.com/ymcatwincities/openy-project/blob/8.1.x/composer.json#L10).
+
+5. Add `"enable-patching": true` to the `extra` section in your `composer.json` See [example](https://github.com/ymcatwincities/openy-project/blob/8.1.x/composer.json#L173).
+
+6. Remove `composer.lock` and `vendor` folder from the project
+
+7. Remove `"replace"` section from your `composer.json`
+
+8. (Optional) If you keep `vendor` folder in your git repository, we recommend to clean up project from `.git` folder inside modules and libraries. To do so
+- Add cleaner script to your project from [OpenY composer package](https://github.com/ymcatwincities/openy-project/blob/8.1.x/scripts/remove_vendor_git_folders.sh). You can just copy it and paste onto your project.
+- [Adjust folders](https://github.com/ymcatwincities/openy-project/blob/8.1.x/scripts/remove_vendor_git_folders.sh#L4) that you would like to cleanup
+- Execute it in `post-install-cmd` and `post-update-cmd`:
+
+    ```
+    "post-install-cmd": [
+        "bash scripts/remove_vendor_git_folders.sh || :"
+    ],
+    "post-update-cmd": [
+        "bash scripts/remove_vendor_git_folders.sh || :"
+    ]
+    ```
+
+9. Run `composer install`
```

Comment:
Also, let's add an example how to add a patch from open Y - https://github.com/propeoplemd/ymcali-openy/blob/master/docroot/composer.json#L118